Runtime/Android/QonversionWrapperAndroid.cs

```diff
@@ -46,6 +46,10 @@ public void SetAdvertisingID()
         {
         }
 
+        public void SetAppleSearchAdsAttributionEnabled(bool enable)
+        {
+        }
```

Comment:
Do we need an empty implementation on android even when we check the platform before calling bridge?

Reply:
Yes, as `QonversionWrapperAndroid` implements `IQonversionWrapper`